### PR TITLE
Terraform Contributions from Keysight Cloud IST

### PR DESCRIPTION
**Pull request by armdupre**
Ixia-c traffic engine deployment on Amazon Web Services with DPDK

### CODE REVIEW

**Comment by dosarudaniel** [comment]
@armdupre , you may want to use the latests `./py/test_*.py` scripts and `*.sh` from https://github.com/open-traffic-generator/otg-examples/pull/69/files

File: public-cloud/aws/ixia-c-dpdk-aws/application/bidirectional.sh
Comment (bortok) [inline public-cloud/aws/ixia-c-dpdk-aws/application/bidirectional.sh:1]: Duplicate of the script from Daniel?

File: public-cloud/aws/ixia-c-dpdk-aws/application/py/test_bidirectional.py
Comment (bortok) [inline public-cloud/aws/ixia-c-dpdk-aws/application/py/test_bidirectional.py:1]: Duplicate of the script from Daniel?

File: public-cloud/aws/ixia-c-dpdk-aws/application/py/test_ipv4_unidirectional.py
Comment (bortok) [inline public-cloud/aws/ixia-c-dpdk-aws/application/py/test_ipv4_unidirectional.py:1]: Duplicate of the script from Daniel?

File: public-cloud/aws/ixia-c-dpdk-aws/application/py/test_throughput_rfc2544_n_flows.py
Comment (bortok) [inline public-cloud/aws/ixia-c-dpdk-aws/application/py/test_throughput_rfc2544_n_flows.py:1]: Duplicate of the script from Daniel?

File: public-cloud/aws/ixia-c-dpdk-aws/application/unidirectional.sh
Comment (bortok) [inline public-cloud/aws/ixia-c-dpdk-aws/application/unidirectional.sh:1]: Duplicate of the script from Daniel?

File: public-cloud/aws/ixia-c-dpdk-aws/application/settings.json
Comment (bortok) [inline public-cloud/aws/ixia-c-dpdk-aws/application/settings.json:1]: Duplicate of the script from Daniel?

File: public-cloud/aws/ixia-c-dpdk-aws/application/rfc2544_test_n_flows.sh
Comment (bortok) [inline public-cloud/aws/ixia-c-dpdk-aws/application/rfc2544_test_n_flows.sh:1]: Duplicate of the script from Daniel?

File: public-cloud/aws/ixia-c-dpdk-aws/application/py/utils/common.py
Comment (bortok) [inline public-cloud/aws/ixia-c-dpdk-aws/application/py/utils/common.py:1]: Duplicate of the script from Daniel?

**Comment by armdupre** [comment]
Will be replaced with a new PR off different branch AWS only.

**Pull request closed without merging**